content/docs/api_reference/policy/v1alpha1.md

```diff
@@ -1,29 +1,23 @@
----
-title: "Policy v1alpha1 API Reference"
-description: "Policy v1alpha1 API reference documentation."
-type: docs
----
-
```

Comment:
Same as before about keeping the header.

And I don't see config/v1alpha2?

Reply:
oh did i not git add

Reply:
ok fixed maybe